jwala-common/src/main/java/com/cerner/jwala/commandprocessor/impl/jsch/JschCommandProcessorImpl.java

```diff
@@ -5,19 +5,20 @@
 import com.cerner.jwala.common.exec.RemoteExecCommand;
 import com.cerner.jwala.common.jsch.JschService;
 import com.cerner.jwala.common.jsch.RemoteCommandReturnInfo;
+import com.cerner.jwala.common.properties.ApplicationProperties;
 import com.cerner.jwala.exception.RemoteCommandFailureException;
 
 import java.io.IOException;
 
 public class JschCommandProcessorImpl implements CommandProcessor {
 
+    private static final String JSCH_SHELL_READ_REMOTE_OUTPUT_TIMEOUT = "jsch.shell.read.remote.output.timeout";
```

Comment:
Don't make this change @Sharvaribarve  is making this change for the entire project